bedrock/settings/static_media.py

```diff
@@ -1610,6 +1611,14 @@
         ),
         'output_filename': 'js/firefox-update-notification-bundle.js',
     },
+    'firefox-update-notification-modal': {
```

Comment:
I'll make the CSS bundle consistent and add `-firefox` to it. This also makes it consistent with the other notification for regular pages.